specification/gedcom-2-data-types.md

```diff
@@ -350,14 +350,14 @@ Version 7.0 only supports the following URLs:
     and should be avoided in datasets that are expected to be shared on the web or with unknown parties,
     but may be appropriate for close collaboration between parties with known similar file structures.
 
-- A URI reference with all of the following:
+- A URI reference, as defined by [RFC 3986](https://www.rfc-editor.org/info/rfc3986), with all of the following:
     - no scheme
     - not beginning with `/` (U+002F)
     - not containing any path segments equal to `..` (U+002E U+002E)
     - not containing a reverse solidus character (U+005C `\`) or `banned` character, either directly or in escaped form
     - no query or fragment
     
-    refers to a **local file**. If the dataset is part of a [GEDZIP file](#gedzip), the URL of the local file is a zip archive filename; otherwise, the URL of a local file is resolved with *base* equal to the directory containing the dataset.
+    refers to a **local file**. If the dataset is part of a [GEDZIP file](#gedzip), the URL of the local file is a zip archive filename; otherwise, the URL of a local file is resolved with *base* equal to the directory containing the dataset.  Note that a URI reference cannot contain non-ASCII characters directly but instead percent-encodes them.
```

Comment:
I disagree with adding the percent-encoded requirement here. While I can see why it could be argued to be technically required by the previous text, I do no think it was an obvious reading of text that explicitly allowed WHATWG URLs too. Could we add something like "or a URL Path as defined by the [WHATWG URL specification](https://url.spec.whatwg.org/)" to not add this constraint on paths? (is that the correct production? I don't know WHATWG as well as I do RFC 3986)